edx_exams/apps/core/models.py

```diff
@@ -18,8 +19,11 @@ class User(AbstractUser):
 
     """
     full_name = models.CharField(_('Full Name'), max_length=255, blank=True, null=True)
+
```

Comment:
Just added this blank line to keep consistent with other formatting in this file.